### ISSUE TITLE
(Documentation) MqttNetGlobalLogger Missing in Current Release (4.0) But Remains in Docs

### ISSUE THREAD

**Issue by iUnknwn** [dotnet/MQTTnet#1366]
### Describe the bug
MqttNetGlobalLogger does not appear to be available, but is still referenced in documentation.

### Which component is your bug related to?
Documentation / Diagnostics.

### Expected behavior
Documentation still appears to reference MqttNetGlobalLogger as the correct way to trace events, but it doesn't look like this class is in the current codebase. 

Is there another class that provides this functionality (global event logging), or is the current way to get events to pass a custom logger to the factory method (as explained in 'Extended Trace')?

**Comment by chkr1011** [comment]
The Wiki only covers version 3. The upcoming version 4 will not have samples in the Wiki at all. Instead please have a look at the "Samples" directory in the repository. There are plenty of examples hot to use the library which will always work with the latest version.

**Comment by iUnknwn** [comment]
I'm happy to update the wiki with better information - I just need clarity on if that class has been replaced with anything. It wasn't included in the 3.1.2 release easier, so it's not just a change in 4.0. 

The sample code seems to show passing a custom event logger to the Mqtt Factory method. If that's the preferred method going forward, I can make a note in the docs. However, before I did that, I wanted to clarify if the global logger was still available by another name, or if its no longer available and the only way to do trace logging is via a custom logger and the factory method.
